python/requirements.txt

```diff
@@ -1,91 +1,35 @@
-# external requirements
```

Comment:
@lastmansleeping Why were there so many unneeded packages listed in the requirements? Did we copy this from another repo?

Reply:
> Replacing my previous comment about `tools/run_docker.sh` after a closer look. With the rearrangement of files, looks like just the README also needs to be updated for locations of the `example_run.sh`. Works when I `cd` into the `python` directory and use `/bin/bash ml4ir/applications/ranking/scripts/example_run.sh`

Will update readme. Thanks!

Reply:
They were not unnecessary really. What I did at ml4ir's inception was take a dump of the packages in my virtualenv packages. So some of the packages are nested dependencies of the packages you see here(for example, keras might be a dependency on tensorflow, which will now get automatically installed anyway). There could have been some from my other repositories as well.

Reply:
That makes sense. Thanks!